examples/tokens/ERC1155ownable.vy

```diff
@@ -0,0 +1,358 @@
+# @version >=0.3.2
+
+# @dev Implementation of ERC-1155 non-fungible token standard
+# @dev ownable, with approval, OPENSEA compatible (name, symbol)
+# @author Dr. Pixel (github: @Doc-Pixel)
+
+############### imports ###############
+from vyper.interfaces import ERC165
```

Comment:
Can you also add an ERC1155 built-in interface? (copy the ERC721) one

Reply:
https://github.com/vyperlang/vyper/blob/master/vyper/builtin_interfaces/ERC721.py

Reply:
Done!